packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -3,23 +3,58 @@ import { DropdownProps, DropdownState } from '../../../../components/Dropdown/Dr
 import { DropdownVariables } from './dropdownVariables'
 import { pxToRem } from '../../../../lib'
 
-const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, DropdownVariables> = {
-  root: (): ICSSInJSStyle => ({}),
+type DropdownPropsAndState = DropdownProps & DropdownState
+
+const transparentColorStyle: ICSSInJSStyle = {
+  backgroundColor: 'transparent',
+  borderColor: 'transparent',
+  borderBottomColor: 'transparent',
```

Comment:
I hate that we have to specify the borderBottomColor as well :\

Reply:
me 2, any ideas?
It's just horrendous how many styles we need to override, this goes to technical debt I'm afraid

Reply:
I saw @kuzhelov closed the corresponded issue for this: https://github.com/stardust-ui/react/issues/237 Are we missing something Roman?

Reply:
oh really, let me try then

Reply:
I removed `borderBottomColor: 'transparent'` and I'm afraid it doesn't work... I shouldn't see the bottom border; see GIF below:

![screen recording 2019-02-08 at 14 12 11](https://user-images.githubusercontent.com/5442794/52480383-d7b6e380-2bab-11e9-8f1e-6f6b0e2a6854.gif)